test/sql/window/test_quantile_window.test_slow

```diff
@@ -0,0 +1,21 @@
+# name: test/sql/window/test_quantile_window.test_slow
+# description: Test windowed QUANTILE at scale
+# group: [window]
+
+require vector_size 512
+
+#statement ok
+#PRAGMA enable_verification
```

Comment:
Is this disabled by accident?

Reply:
Yes. More precisely, I was timing queries and wanted to remove overhead. I'll restore it.

Reply:
After checking, I have removed it because it dramatically increases the run time.

Reply:
Upon review, this test file was an accident and should be removed. The other test file is more complete.